sim/src/main/scala/spinal/sim/VerilatorBackend.scala

```diff
@@ -644,7 +645,7 @@ JNIEXPORT void API JNICALL ${jniPrefix}disableWave_1${uniqueId}
         FileUtils.copyDirectory(workspaceCacheDir, workspaceDir)
       }
 
-      genWrapperCpp()
+      genWrapperCpp(verilatorVersionDeci > BigDecimal("4.034"))
```

Comment:
I'm curious, how have you found this value `4.034`?

Reply:
First tagged commit after the time precision function was added to Verilator

Reply:
So isn't is `>=`?